### PR TITLE
Add basic CONTRIBUTING.md file

### PR DESCRIPTION
**Pull request by Lukasa**
It's beginning to get frustrating to have to keep dealing with questions on this issue tracker. Let's see if we can dissuade them.

### CODE REVIEW

**Comment by adrianmoisey** [comment]
That looks great!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+     Failing that, let us know what you're doing, how often it happens, what
+     environment you're using, etc. Be thorough: it prevents us needing to ask
+     further questions.
+   - Tell us **what you expected to happen**. When we run your example code,
+     what are we expecting to happen? What does "success" look like for your
+     code?
+   - Tell us **what actually happens**. It's not helpful for you to say "it
+     doesn't work" or "it fails". Tell us *how* it fails: do you get an
+     exception? A hang? A non-200 status code? How was the actual result
+     different from your expected result?
```
Comment (sigmavirus24) [inline CONTRIBUTING.md:49]: Also what version of requests you're using and _how_ you installed it.

**Comment by sigmavirus24** [comment]
One addition to what makes a good bug report for us.

**Comment by Lukasa** [comment]
Updated. =)

**Comment by sigmavirus24** [comment]
I'm not confident people will read this before filing a bug, but we can at least point them to it when closing a bug now.

**Comment by Lukasa** [comment]
More importantly, because GitHub prompts people to look at it first we can be sure that anyone who doesn't do what this says definitely didn't care.

**Pull request merged**